composer.json

```diff
@@ -36,7 +36,9 @@
         "jackalope/jackalope-doctrine-dbal": "^1.0",
         "php-mock/php-mock": "^1.0",
         "predis/predis": "^0.8 || ^1.0",
-        "symfony/phpunit-bridge": "^3.3.12 || ^4.0"
+        "symfony/console": "^2.8 || ^3.2 || ^4.0",
+        "symfony/framework-bundle": "^2.8 || ^3.2 || ^4.0",
```

Comment:
Why don't we need this on the require section?

Reply:
On master it is on require-dev section already. Added here: https://github.com/sonata-project/SonataCacheBundle/pull/161. This is only used for the commands. Not sure if the intention is to have commands only optionals. With this we do not really change what gets installed, because other dependencies on require-dev already use framework-bundle.

We can maybe think if this should be required o require-dev on the pull request where we add commands as a services.